INSTRUCTION.md

```diff
@@ -0,0 +1,63 @@
+# INSTRUCTION.md
+
+## 1. Як запускати контейнер MySQL
+
+Щоб запустити контейнер MySQL, виконайте наступну команду в терміналі:
+
+docker run -d --name mysql-container \
+  -e MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD} \
+  -e MYSQL_DATABASE=${MYSQL_DATABASE} \
+  -e MYSQL_USER=${MYSQL_USER} \
+  -e MYSQL_PASSWORD=${MYSQL_PASSWORD} \
+  -v mysql_data:/var/lib/mysql -p 3306:3306 mysql-local:1.0.0
+
+# Ця команда:
+
+Створює контейнер з ім'ям mysql-container.
+Встановлює пароль для користувача root як rootpassword.
+Створює базу даних app_db та користувача app_user з паролем 1234.
+Прив'язує том mysql_data для зберігання даних.
+Відкриває порт 3306 для доступу до MySQL.
+
+# Щоб перевірити, чи працює контейнер, скористайтесь командою:
+
+docker ps
+
+# 2. Як запускати додаток і підключати до бази даних
+Переконайтесь, що контейнер MySQL працює (як описано вище).
+Оновіть налаштування підключення до MySQL в файлі settings.py вашого проєкту Django:
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'mysql.connector.django',
+        'NAME': 'app_db',
+        'USER': 'app_user',
+        'PASSWORD': '1234',
+        'HOST': 'localhost',  # Змінити на IP контейнера MySQL пізніше.
```

Comment:
Ensure that the `HOST` value is updated with the actual IP address of the MySQL container. Using 'localhost' will not work when the database is running in a separate container.